fgpyo/sam/__init__.py

```diff
@@ -547,6 +547,48 @@ def length_on_target(self) -> int:
         """Returns the length of the alignment on the target sequence."""
         return sum([elem.length_on_target for elem in self.elements])
 
+    def get_alignment_offsets(self, reverse: bool = False) -> Tuple[int, int]:
```

Comment:
The name is quite ambiguous given that we're dealing with an alignment between a query and a target.  How about one of the following:

`get_query_alignment_offsets`
`get_query_alignment_bounds`
`get_query_alignment_range`

Also: consider returning a `range` object?

Reply:
you mean a python range ?

Reply:
neat idea!

Reply:
Functions prefixed with `get_` are usually reserved for true property [getters](https://docs.python.org/3/library/functions.html#property.getter).

Can we drop the prefix too?


```suggestion
    def alignment_offsets(self, reverse: bool = False) -> Tuple[int, int]:
```

https://google.github.io/styleguide/pyguide.html#315-getters-and-setters